src/Rector/ClassConstFetch/ConsoleExceptionToErrorEventConstantRector.php

```diff
@@ -57,10 +59,13 @@ public function refactor(Node $node): ?Node
     {
         if ($node instanceof ClassConstFetch && (
             $this->isObjectType($node->class, $this->consoleEventsObjectType) &&
-            $this->isName($node->name, 'EXCEPTION')
+            $this->isName($node->name, EXCEPTION::class)
```

Comment:
is seems new bug on `StringClassNameToClassConstantRector`, need to skip it for now.

Reply:
https://github.com/rectorphp/rector-symfony/pull/236